### PR TITLE
feat(profile): upgrade playground to next version of the SDK

### PR DESCRIPTION
**Pull request by Nuhvi**
Switches to the latest version of the SDK, and add support for the seeder.

Added support for both p2wpkh and zero sats ln invoice... but the playground probably creates manner and non p2wpkh addresses, so it is never valid.

However, feel free to pay for this slashtag, it should always be resolvable (from seeder):
`slash:4c71h4rmdpddqroj7freshg34h59wqytde9nm654geide7yt3jqo`

https://user-images.githubusercontent.com/40009100/190394506-83a2be74-1d97-416e-8857-cd61b94da10f.mov

### CODE REVIEW

File: src/components/ContactsList.tsx [hunk 1]
```diff
@@ -56,11 +53,12 @@ const ContactsList = ({
...
 }): ReactElement => {
-	const { contacts } = useSlashtagsContacts();
-	const { url: myProfileURL, profile } = useSelectedSlashtag();
+	const contacts = useSlashtags().contacts;
+	const { url: myProfileURL } = useSelectedSlashtag();
 
 	const filteredContacts = useMemo(() => {
 		return Object.values(contacts)
+			.sort((a, b) => (a.name > b.name ? 1 : -1))
 			.sort((a, b) => (a.name > b.name ? 1 : -1))
```
Comment (limpbrains) [inline src/components/ContactsList.tsx]: double sort

File: src/screens/Profile/ProfileEdit.tsx [hunk 2]
```diff
@@ -131,7 +137,7 @@ export const ProfileEdit = ({ navigation }): JSX.Element => {
 							bottomSheet={true}
 							label="Label"
 							value={addLinkForm.label}
-							placeholder="For example ‘Website’"
+							placeholder="For example �Website�"
```
Comment (limpbrains) [inline src/screens/Profile/ProfileEdit.tsx]: this need to be reverted. On my iOS simulator it can't render these chars

**Review by limpbrains — CHANGES_REQUESTED** [review]

**Review by limpbrains — APPROVED** [review]

**Review by Jasonvdb — APPROVED** [review]

**Comment by pwltr** [comment]
This one is a bit faulty. An unhandled promise rejection is making my simulators crash. Something in the `<SlashtagsProvider />` component. Looking into it.

**Comment by limpbrains** [comment]
> This one is a bit faulty. An unhandled promise rejection is making my simulators crash. Something in the `<SlashtagsProvider />` component. Looking into it.

wiping wallet data solved this issue for me

**Pull request merged**